src/Tizen.NUI/src/public/Layouting/LayoutGroup.cs

```diff
@@ -562,5 +562,15 @@ internal static void OnChildPropertyChanged(Binding.BindableObject bindable, obj
             View view = bindable as View;
             view?.Layout?.RequestLayout();
         }
+
+        internal static Binding.BindableProperty.ValidateValueDelegate ValidateEnum(int enumMin, int enumMax)
+        {
+
+            return (Binding.BindableObject bindable, object value) =>
+            {
+                int @enum = (int)value;
+                return enumMin <= @enum && @enum <= enumMax;
+            };
+        }
     }
 }
```

Comment:
could you add new line at the end of file?
reference) https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline